react/Dropdown/Dropdown.demo.js

```diff
@@ -1,8 +1,8 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
 import Dropdown from './Dropdown';
 import styles from './Dropdown.less';
+import * as sketch from './Dropdown.sketch';
```

Comment:
It looks like you only have the one export, I'd probably just use the name `symbol` directly.

Reply:
It's imported this way so that we can pass the `sketch` object as part of the demo spec further down the file, which is what renders the sketch content to the page.

Reply:
So the shape of that object is something like:
```javascript
{
  symbols: {
    [key: string]: React$Node
  }
}
```

Just trying to understand it, all makes sense to me and you know the most about this code.

Reply:
Yep, exactly 👍